src/tools/gophish_import.py

```diff
@@ -69,19 +68,19 @@ def load_landings(api, assessment):
                 break
             except Error as e:
                 if e.message == "Page name already in use":
-                    logging.warning(f"{e}. Finding with previously loaded page.")
+                    logging.warning("%s. Finding with previously loaded page.", e)
                     old_pages = api.pages.get()
                     for old_page in old_pages:
                         if old_page.name == new_page.name:
-                            logging.debug(f"Deleting Page with ID {old_page.id}")
+                            logging.debug("Deleting Page with ID %i", old_page.id)
```

Comment:
Is the ID an integer (`%d`)?  Or is it a string (`%s`)?
```suggestion
                            logging.debug("Deleting Page with ID %d", old_page.id)
```

Reply:
See aa93585